coderadar-core/src/main/java/io/reflectoring/coderadar/vcs/port/driven/GetRawCommitContentPort.java

```diff
@@ -3,6 +3,7 @@
 import io.reflectoring.coderadar.vcs.UnableToGetCommitContentException;
 import java.util.HashMap;
 import java.util.List;
+import org.apache.commons.lang3.tuple.Pair;
```

Comment:
We could use the `Pair` from spring data and remove the `commons-lang3` dependency.